src/Models/Site.php

```diff
@@ -91,9 +91,9 @@ public function exceptionLogGroups(): HasMany
         return $this->hasMany(ExceptionLogGroupRepository::resolveModelClass());
     }
 
-    public function serverMetrics(): HasMany
+    public function serverMetrics(): HasOne
     {
-        return $this->hasMany(ServerMetric::class);
+        return $this->hasOne(ServerMetric::class)->latestOfMany();
```

Comment:
The previous relationship was good, not needed to deleted. 

Add a new method:

```php
public function latestServerMetric(): HasOne
{
    return $this->hasOne(ServerMetric::class)->latestOfMany();
}
```